cabal-install/Distribution/Client/List.hs

```diff
@@ -327,9 +327,9 @@ showPackageDetailedInfo :: PackageDisplayInfo -> String
 showPackageDetailedInfo pkginfo =
   renderStyle (style {lineLength = 80, ribbonsPerLine = 1}) $
    char '*' <+> disp (pkgName pkginfo)
-            <>  maybe empty (\v -> char '-' <> disp v) (selectedVersion pkginfo)
+            Disp.<> maybe empty (\v -> char '-' Disp.<> disp v) (selectedVersion pkginfo)
```

Comment:
This kinda makes me long for OCaml's local module open feature.

Reply:
:-) yeah...

fwiw, I used the module-qualifier strategy as it's more robust than having to use imports w/ `hiding`...